### ISSUE TITLE
conference and event accessibility policy

### ISSUE THREAD

**Issue by deborahgu** [w3c/PWETF#6]
The W3C should have an accessibility policy for conferences, workshops, meetings, and any other live events. It should cover:

* hotel selection
* venue selection (including bathrooms, ramps to speaker spaces, etc.)
* documenting 
  * transport 
  * accessible routes
  * outlets for charging powerchairs, ventilation machines, etc.
* speaker policies
  * hand-held microphones for questioners
  * pre-providing digital materials
  * accessible digital materials for within the conference
  * describing slides while talking
  * whether support is available for speakers to help make accessible materials
* Hearing assistance:
  * CART or other transcription
  * sign interpretation
  * induction loops
* a fragrance / smoke policy
* having a low-stimulation space
* color communication badges
* service animals
* funding, space, and meeting admittance for aides 
* dietary needs
* whether remote/virtual attendance is allowed
* whether there is funding available (whether for speakers, event organizers, or attendees) to help support any of this, and if so, how to apply for it.
* checklist, contact people for help, and documentation for meeting organizers
* accessibility point of contact both before and during the event

Good resources:

* [ACM's conference guide for the special interest group on accessible computing](http://www.sigaccess.org/welcome-to-sigaccess/resources/accessible-conference-guide/)
* [color communication badges](https://autisticadvocacy.org/wp-content/uploads/2014/02/ColorCommunicationBadges.pdf)
* [list of conferences and cons with accessibility policies](http://geekfeminism.wikia.com/wiki/Cons_with_Accessibility_Policies) (Starts with cons, but with professional conferences at the bottom.)

**Comment by jeffjaffe** [comment]
Please work with @alexandralacourba who is responsible for events for W3C.

**Comment by brewerj** [comment]
I will assess our resources on the different items listed here and report back, to sort out next steps.
On communication accommodations, Dom and I are working on a proposal. 
For presentations and events, this resource is available:
https://www.w3.org/WAI/teach-advocate/accessible-presentations/

**Comment by TzviyaSiegman** [comment]
@brewerj and @alexandralacourba can this issue be shifted to the Team? Should it remain open for reference?

**Comment by TzviyaSiegman** [comment]
https://www.w3.org/TR/remote-meetings/ covers remote meetings.
